File: Python/p10300.py
tc = int(input())
for i in range(tc):
    banyak = int(input())
    res = 0.0
    for j in range(banyak):
        luas, binatang, poin = map(int, input().split())
        hasil = (luas/binatang) * poin * binatang
        res += hasil

    print("{:.0f}".format(res))

File: Java/p10300.java
import java.util.Scanner;

public class p10300 {
	public static void main (String [] args) {
		Scanner input = new Scanner(System.in);
		int tc = input.nextInt();
		for (int i = 0; i < tc; i++) {
			int banyak = input.nextInt();
			double res = 0;
			for (int j = 0; j < banyak; j++) {
				int luas = input.nextInt();
				int binatang = input.nextInt();
				int poin = input.nextInt();
				double hasil = ((luas*1.0)/(binatang*1.0)) * (poin*1.0) * (binatang*1.0);
				res += hasil;
			}
			
			System.out.printf("%.0f\n", res);
		}
	}
}